hs_core/resourcemap_urls.py

```diff
@@ -0,0 +1,14 @@
+__author__ = 'pabitra'
```

Comment:
I am not one to know this.  If it is std practice it is fine.  A comment in the code to say you wrote this is fine, but this looked like an assignment so I asked the question.